test/models/UserModel/test.js

```diff
@@ -0,0 +1,322 @@
+        skip: user,
+      });
+      expect(error).to.be.exist;
+    });
+  });
+  describe('On getFollowings', async () => {
+    let name;
+
+    beforeEach(async () => {
+      name = faker.name.firstName();
+      await UsersFactory.Create({ name });
+    });
+    it('Should return an array', async () => {
+      const { users } = await UserModel.getFollowings({ name, skip: 0, limit: 10 });
+      expect(Array.isArray(users)).to.be.true;
+    });
+    it('Should return an empty array', async () => {
+      const { users } = await UserModel.getFollowings({ name, skip: 0, limit: 10 });
+      expect(_.isEmpty(users));
+    });
+    it('Should return correct users array', async () => {
+      name = faker.name.firstName();
+      const followings = _.random(5, 15);
+      const followingsArray = new Array(followings);
+      await UsersFactory.Create({ name, users_follow: followingsArray });
+      const { users } = await UserModel.getFollowings({ name, skip: 0, limit: followings });
+      expect(users.length).to.be.eq(followings);
+    });
+  });
+  describe('On search', async () => {
+    let user;
+    let options;
+    beforeEach(async () => {
+      await dropDatabase();
+      user = await UsersFactory.Create();
+      options = {
+        string: user.name,
+        skip: 0,
+        limit: 10,
+        notGuest: false,
+      };
+    });
+    it('Should return a correct user', async () => {
+      const { users } = await UserModel.search(options);
+      expect(users[0].name).to.be.eq(user.name);
+    });
+    it('Should return an array', async () => {
+      const { users } = await UserModel.search(options);
+      expect(Array.isArray(users)).to.be.true;
+    });
+    it('Should check that the error exists', async () => {
+      const { error } = await UserModel.search({
+        test: _.random(0, 50),
+        host: faker.random.string(),
+        skip: user,
+      });
+      expect(error).to.be.exist;
+    });
+  });
+  describe('On find', async () => {
+    let name;
+    let user;
+    let userAlias;
+    let condition;
+    let usersCount;
+    beforeEach(async () => {
+      await dropDatabase();
+      name = faker.name.firstName();
+      userAlias = faker.lorem.word();
+      user = await UsersFactory.Create({ name, alias: userAlias });
+      usersCount = _.random(1, 10);
+      for (let iter = 0; iter < usersCount; iter++) {
+        await UsersFactory.Create();
+      }
+      condition = { alias: userAlias };
+    });
+    it('Should return an array', async () => {
+      const { usersData } = await UserModel.find({
+        condition,
+        select: { alias: user.alias },
+        skip: 0,
+      });
+      expect(Array.isArray(usersData)).to.be.true;
+    });
+    it('Should return correct user', async () => {
+      const { usersData } = await UserModel.find({
+        condition,
+        select: { alias: user.alias },
+        skip: 0,
+      });
+      expect(usersData[0].alias).to.be.eq(userAlias);
```

Comment:
you can make a nested describe method, in which, in a before hook, call the search method and in it cases, just do an expect